tests/acceptance/00_basics/ifelapsed_and_expireafter/timed/defaults.cf

```diff
@@ -30,9 +30,9 @@ bundle agent init
   meta:
       # Backgrounding doesn't work properly on Windows or with fakeroot.
       # In addition, RHEL4 seems to have some problem killing the test
-      # afterwards, and HP-UX and Solaris 9 and 10 are too slow for timing
-      # to be accurate.
-      "test_skip_needs_work" string => "windows|using_fakeroot|redhat_4|hpux|sunos_5_9|sunos_5_10";
+      # afterwards, and HP-UX and Solaris are too slow for timing
+      # to be accurate. Also, this test randomly fails on SUSE 11 and 12.
```

Comment:
Please do not introduce (more) lies -- Solaris 11 is *not* too slow for timing to be accurate. There are genuine problems in our code or in LMDB.